api/src/main/resources/db/migration/v2025.01.20_14.00__engine_run_task.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS pipeline.engine_run_task (
+	task_id         TEXT        NOT NULL,
+	task_name       TEXT,
+	task_key        TEXT,
+	task_group      TEXT,
+	parent_id       TEXT,
+	engine_type     TEXT        NOT NULL,
+	status          TEXT        NOT NULL,
+	engine_run_id   TEXT,
+	engine_run_name TEXT,
+	data            JSONB,
+	start_date      TIMESTAMP WITH TIME ZONE,
+	end_date        TIMESTAMP WITH TIME ZONE,
+	run_id          BIGINT      NOT NULL REFERENCES pipeline.pipeline_run(run_id),
+	duration        BIGINT,
+	CONSTRAINT engine_run_task_constrain UNIQUE(task_id, engine_type)
```

Comment:
this pair can't be unique because for each nf run task_id will start from 1. Probably for primary key we need artificial id